### PR TITLE
Bump the x/net dependency in git-lfs to mitigate a GHSA.

### PR DESCRIPTION
**Pull request by dlorenc**
I've built a git image with this and verified it fixes the GHSA in a grype scan.

### CODE REVIEW

**Review by imjasonh — APPROVED** [review]

**Comment by imjasonh** [comment]
Are there other packages that depend on x/net that should also be updated?

File: git-lfs.yaml [hunk 1]
```diff
@@ -36,6 +36,9 @@ pipeline:
 
       # mitigate CVE-2022-27191
       go get golang.org/x/crypto/ssh
+
+      # mitigate GHSA-vvpx-j8f3-3w6h
+      go get golang.org/x/net@v0.7.0
```
Comment (imjasonh) [inline git-lfs.yaml:41]: Do we need a `secfixes`/`advisory` entry for this? And the above?

Comment (luhring) [inline git-lfs.yaml:41]: Is there a risk to using the version here? Will this cause us to accidentally downgrade in the future?

Comment (dlorenc) [reply git-lfs.yaml:41]: I don't think we do secfixes for non-CVEs, but I could be mistaken.

Comment (dlorenc) [reply git-lfs.yaml:41]: I added a comment to the version number to tell the next person to try removing all of these. You're right we'd risk downgrading, but IMO that's better than getting a potentially new, unpinned version of the package each time we build.

Comment (luhring) [reply git-lfs.yaml:41]: Secfixes/advisories are only for vulns for `apk` packages, not `go` packages (i.e. modules). So no need to add anything here!

**Review by kaniini — APPROVED** [review]

**Pull request merged**